perf/bchar_perf.ml

```diff
@@ -2,21 +2,19 @@ open Perf
 
 let _ = 
   runPerfTest "getDigit" (fun _ ->
-      let c = Bchar.getDigit '1' in
-      match c with
-      | Some _ -> ()
-      | None -> ()
-    );
+    let c = Bchar.getDigit '1' in
+    match c with
+    | Some _ -> ()
+    | None -> ()
+  );
   runPerfTest "getDigitOrRaise" (fun _ -> 
-      let _ = (try 
-                 Bchar.getDigitOrRaise '1'
-               with
-               | Failure _ -> 1) in ();
-    );
+    let _ = (try Bchar.getDigitOrRaise '1'
+      with
+      | Failure _ -> 1) in ();
```

Comment:
This doesn't look very good.

Either
```ml
let _ = (
  try
    Bchar.getDigitOrRaise '1'
  with
  | Failure _ -> 1) in ();
```
or
```ml
let _ = (
  try Bchar.getDigitOrRaise '1'
  with Failure _ -> 1) in ();
```
or perhaps using `begin`/`end` instead of parens would be better